Dockerfile.assisted-installer

```diff
@@ -1,8 +1,16 @@
-FROM quay.io/centos/centos:centos8
-copy build/installer /
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
+ENV GOFLAGS=-mod=mod
+WORKDIR /go/src/github.com/openshift/assisted-installer
+COPY . .
+RUN make installer
+
+FROM registry.svc.ci.openshift.org/ocp/4.7:base
 ARG GIT_REVISION
-LABEL "git_revision"=${GIT_REVISION}
 ARG QUAY_TAG_EXPIRATION
+
+COPY --from=builder /go/src/github.com/openshift/assisted-installer/build/installer /usr/bin/installer
+COPY --from=builder /go/src/github.com/openshift/assisted-installer/deploy/assisted-installer-controller /assisted-installer-controller
+
+LABEL "git_revision"=${GIT_REVISION}
```

Comment:
iirc, Prow doesn't work well with Dockerfile arguments.
You can Remove the labels and pass them as part of the build process, without arguments.

For example on [assisted-service](https://github.com/openshift/assisted-service/blob/7aa360c9ac4ad66c1afddaf20b18a484a1070268/Makefile#L31)